File: Python/tests/test_ts_par6_trial.py
import pytest

from hbayesdm.models import ts_par6_trial


def test_ts_par6_trial():
    _ = ts_par6_trial(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_ts_par6_trial.R
context("Test ts_par6_trial")
library(hBayesDM)

test_that("Test ts_par6_trial", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(ts_par6_trial(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
